src/Auth/Auth.js

```diff
@@ -92,6 +92,14 @@ function isAsync () {
   return false;
 }
 
+function acquireTokens (callback) {
+  initProviderIfNull();
+  if (currentProvider === undefined) {
+    return false;
```

Comment:
Could we return `undefined` instead of `false` ? A boolean does not make much sense here